modules/cli/src/main/scala/scala/cli/exportCmd/JsonProject.scala

```diff
@@ -0,0 +1,92 @@
+package scala.cli.exportCmd
+
+import com.github.plokhotnyuk.jsoniter_scala.core.{JsonValueCodec, WriterConfig, writeToStream}
+import com.github.plokhotnyuk.jsoniter_scala.macros.{CodecMakerConfig, JsonCodecMaker}
+import coursier.Dependency
+import coursier.util.Artifact
+
+import java.nio.charset.StandardCharsets
+
+import scala.build.options.ConfigMonoid
+import scala.cli.util.SeqHelpers.*
+import scala.reflect.NameTransformer
+import scala.util.{Properties, Using}
+
+final case class JsonProject(
+  projectName: Option[String] = None,
+  scalaVersion: Option[String] = None,
+  platform: Option[String] = None,
+  scalacOptions: Seq[String] = Nil,
+  scalaCompilerPlugins: Seq[String] = Nil,
```

Comment:
Aren't these supposed to be dependencies too? (so something like `Seq[ExportDependencyFormat]`)